aiida_common_workflows/workflows/eos.py

```diff
@@ -143,8 +148,16 @@ def inspect_eos(self):
             return self.exit_codes.ERROR_SUB_PROCESS_FAILED.format(cls=self.inputs.sub_process_class)  # pylint: disable=no-member
 
         for index, child in enumerate(self.ctx.children):
-            volume = child.outputs.relaxed_structure.get_cell_volume()
+            try:
+                volume = child.outputs.relaxed_structure.get_cell_volume()
+            except exceptions.NotExistent:
+                volume = self.ctx.structures[index]
+
             energy = child.outputs.total_energy.value
+
             self.report(f'Image {index}: volume={volume}, total energy={energy}')
             self.out(f'structures.{index}', child.outputs.relaxed_structure)
```

Comment:
You forgot the call here (well, I forgot to point it out), but it seems to work! Nice idea!